source/portable/NetworkInterface/STM32Hxx/NetworkInterface.c

```diff
@@ -183,17 +181,16 @@ static size_t uxGetOwnCount( ETH_HandleTypeDef * heth );
  * They can be defined as static because the function addresses
  * addresses will be stored in struct NetworkInterface_t. */
 
-static BaseType_t xNetworkInterfaceInitialise( NetworkInterface_t * pxInterface );
-
-static BaseType_t xNetworkInterfaceOutput( NetworkInterface_t * pxInterface,
-                                           NetworkBufferDescriptor_t * const pxBuffer,
-                                           BaseType_t xReleaseAfterSend );
+static BaseType_t xSTM32H_NetworkInterfaceInitialise( NetworkInterface_t * pxInterface );
 
-static BaseType_t xGetPhyLinkStatus( NetworkInterface_t * pxInterface );
+static BaseType_t xSTM32H_NetworkInterfaceOutput( NetworkInterface_t * pxInterface,
+                                                  NetworkBufferDescriptor_t * const pxBuffer,
+                                                  BaseType_t xReleaseAfterSend );
 
-NetworkInterface_t * pxFillInterfaceDescriptor( BaseType_t xEMACIndex,
-                                                NetworkInterface_t * pxInterface );
+static BaseType_t xSTM32H_GetPhyLinkStatus( NetworkInterface_t * pxInterface );
 
+NetworkInterface_t * pxSTM32Hxx_FillInterfaceDescriptor( BaseType_t xEMACIndex,
```

Comment:
You're using two different prefixes here: `pxSTM32Hxx_FillInterfaceDescriptor()` and `xSTM32H_NetworkInterfaceOutput`.
I prefer the second one: `STM32H_`.

Reply:
Done, thanks